openmrs-19/src/main/java/org/motechproject/openmrs19/service/impl/OpenMRSPersonServiceImpl.java

```diff
@@ -91,8 +91,25 @@ public Person createPerson(String configName, String givenName, String familyNam
 
     @Override
     public Person updatePerson(String configName, Person person) {
+        Validate.notEmpty(person.getUuid(), "Person uuid cannot be null");
         try {
             Config config = configService.getConfigByName(configName);
+
+            Person fetchedPerson = personResource.getPersonById(config, person.getUuid());
+
+            //Update is splited on separately update address, names and general information, because of

```

Comment:
please correct this comment. It should be 'Updating address and name of person must be done separately'